lms/templates/certificates/_certificate-supplemental.html

```diff
@@ -0,0 +1,36 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! import mako.runtime %>
+<% mako.runtime.UNDEFINED = '' %>
+
+<aside class="content content-supplemental" role="complimentary"  id="validation-info">
+  <div class="content-supplemental-about">
+    <h2 class="title">${company_about_title}</h2>
+    <div class="copy">
+      <p>${company_about_description}</p>
+    </div>
+    <ul class="list list-actions">
+      <li class="item item-action">
+        <a class="action action-primary action-moreinfo" href="${company_about_url}">${company_about_urltext}</a>
+      </li>
+    </ul>
+  </div>
+
+  <!-- TODO: determine if Open edX needs verification techniques and if so how to support? -->
```

Comment:
Thanks for calling this out specifically. I think we should remove the section that has verification techniques for certificates now that we are moving to web view certificates. In this context, this supplemental section isn't useful. If we'd like to take care of this in this PR that works for me, or we can add it to the list of things to cover in a future PR.

NOTE - Removing this would also let us remove all the other images associated with the other pages related to this validation workflow.